apps/examples/wifi_manager/wifi_manager_sample/README.txt

```diff
@@ -1,5 +1,5 @@
-examples/wifi_manager_sample
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+examples/wifi_manager/wifi_manager_sample
```

Comment:
Do you think only wm_sample is sufficient under wifi_manager directory ?

Reply:
Do you mean other examples should be located in wifi_manager? Please share it to me if I missed

Reply:
I agree with your opinion. `wm_` is enough to present the test name because of parent directory.
While I modified, I found that all names of things like command name, file names in this folder have `wm_test` and `sample` is not matched so I modified this name to `wm_test`.
Please check my update again. Thank you!